tests/pytest.ini

```diff
@@ -0,0 +1,7 @@
+[pytest]
+minversion = 3.0
+addopts =
+    # show summary of skipped tests
+    -rs
```

Comment:
This removed `w` from what was in the CMake version.  I assume that was intentional (I can't even find `w` in the pytest manpage; was it a deprecated option?), but just checking.